exercises/poker/poker.exs

```diff
@@ -0,0 +1,16 @@
+defmodule Poker do
+  @doc """
+  Given a list of poker hands, return a list containing the highest scoring hand.
+
+  If two or more hands tie, return the list of tied hands in the order they were received.
+
+  Aces can be used in low (A 2 3 4 5) or high (10 J Q K A) straights, but do not count as
+  a high card in the former case.
+
+  For example, (A 2 3 4 5) will lose to (2 3 4 5 6).
+  """
```

Comment:
No problem, I completely understand, and am about to push some changes to fix the problems you mentioned, should be much clearer now.

I have the same problem with the Diffie-Hellman Exchange one I'm working on. Literally almost nothing in the explanation of the problem to be implemented.